provider/cmd/pulumi-resource-command/schema.json

```diff
@@ -191,8 +191,8 @@
                     "$ref": "#/types/command:remote:Connection"
                 },
                 "localPath": {
-                    "description": "The path of the file to be copied.",
-                    "type": "string"
+                    "description": "The file/folder to be copied.\nIf the input is an Asset, it will be interperted as the contents of a file.\nIf the input is an Archive, it will be interperted as the contents of a folder.\nIf the input is a string, it will be interpreted as the path to a file or folder.\nBecause assets and archives are understood by the Pulumi model, they use should be\nprefered to raw paths.",
```

Comment:
The main reason an Asset or Archive could be more useful than a path string in general is that the asset/archive computes the hash, which allows identifying whether the file contents have changed, and thus triggering an update/replace.  Is that accomplished here?  If so, might be good to call that out as the reason for preferring Asset/Archive?